App/AuthProvider.js

```diff
@@ -3,223 +3,119 @@
  * Licensed under the MIT License.
  */
 
-const {
-  PublicClientApplication,
-  CryptoProvider,
-} = require("@azure/msal-node");
-const { BrowserWindow } = require("electron");
-const CustomProtocolListener = require("./CustomProtocolListener");
-const { msalConfig, REDIRECT_URI } = require('./authConfig')
+const { PublicClientApplication } = require('@azure/msal-node');
+const { msalConfig } = require('./authConfig');
+const open = require('open');
```

Comment:
maintainers of the open package suggests:

> If you need this for Electron, use [shell.openPath()](https://www.electronjs.org/docs/api/shell#shellopenpathpath) instead.

let's look into removing this dependency and use the electron way